### PR TITLE
Feature/column-sizing

### PR DESCRIPTION
**Pull request by noahmanger**
This includes various and sundry improvements to the way that data tables present data. Combined with the style changes in https://github.com/18F/fec-style/pull/412 , it should help make our data tables more useful and more legible. 

- Re-arranges columns throughout for a more readable order (see my note about the rules for organizing columns below)
- Disables the `autoWidth` option on all data tables
- The style change turns off truncating cells, so the full cell content will always be visible
- Applies classes to columns to control sizing in CSS rather than in the Javascript. These size classes are then applied to try to constrain the widths of cells and force others to be larger
- Improves the logic around which columns are shown when the panel is open
- Only shows the "independent expenditures" column on the single committee filing tab if it's the kind of committee that makes IEs
- Fixes a bug where all filing table rows had the expand button, when only certain ones should 

For example:
image

image

image

Resolves https://github.com/18F/openFEC-web-app/issues/1304
Depends on https://github.com/18F/fec-style/pull/412

I still definitely want to pair with @jenniferthibault on this, but wanted to get the PR in before the weekend.

## Thoughts on organizing rules
As I was working on this I started sketching out a set of rules that could hopefully guide further decisions around how to present columns. Here's a draft:
- Number columns should always be right-aligned and should always be positioned furthest to the right, which is more readable when numbers are right-aligned and also allows for easier scanning
- If a row is a transaction, the amount of the transaction should always be furthest to the right and the committee reporting the transaction should always be furthest to the left (notably changes receipt and disbursement column order)
- On desktop, when a panel is open, the first 2-3 columns should be the only ones shown in order to prevent a jarring experience of columns appearing to be out of order
- On tablet, when a panel is open, only the 1-2 columns should be shown
- In general, the first column should be the candidate or committee who made transaction or submitted a report, on order to have a clear "subject" to the row
- Any table with chronological data should be sorted by date by default

These aren't hard and fast rules, but I think should only be broken with good reason.

### CODE REVIEW

**Comment by noahmanger** [comment]
@jenniferthibault raises a good point default sorting by date: are there times when tables should be sorted by amount by default? I'm not sure, but could definitely see that being the case.

**Comment by noahmanger** [comment]
This also now sets a max for the tables with bars in the cells that persists after paginating, so the bars decrease in size as you page through.

Resolves https://github.com/18F/openFEC/issues/1704

And with that this is ready for code review. @jenniferthibault gave the 👀  check. We still have somewhat of a lingering issue with figuring out a better solution for preventing column titles from overlapping with sort icons, but that doesn't need to hold this up.

File: static/js/modules/columns.js [hunk 1]
```diff
@@ -117,104 +115,99 @@ var committees = [
...
-  dateColumn({data: 'transaction_date', className: 'min-tablet hide-panel-tablet'}),
+  currencyColumn({data: 'transaction_amount', className: 'min-tablet hide-panel column--med column--number'}),
+  dateColumn({data: 'transaction_date', className: 'min-tablet hide-panel column--med'}),
   modalTriggerColumn
 ];
 
 var disbursements = [
   {
-    data: 'recipient_name',
+    data: 'committee',
```
Comment (LindsayYoung) [inline static/js/modules/columns.js]: just double checking the change `recipient_name` should be replaced by `committee`

File: static/js/pages/committee-single.js [hunk 2]
```diff
@@ -136,15 +136,21 @@ var occupationColumns = [
   }
 ];
 
+var columnKeys =   [
+  'pdf_url', 'amendment_indicator', 'receipt_date', 'coverage_end_date',
+  'total_receipts', 'total_disbursements', 'modal_trigger'
+];
+
+if (context.showIndependentExpenditures) {
```
Comment (LindsayYoung) [inline static/js/pages/committee-single.js]: So smart

**Pull request merged**